contracts/view.html

```diff
@@ -92,8 +93,25 @@ <h1 class="page-header">State of California Contracts</h1>
     <footer>
       <ol class="footnote-listing">
         {% for footnote in site.data.contracts.footnotes %}
-          <li id="fn-{{ footnote[0] }}">
-            {{ footnote[1] }}
+          {% assign products = "" | split: "" %}
+          {% assign footnote_id_str = footnote[0] | append: "" %}
+          
+          {% comment%}
+          Build up product data-attribute string for each footnote 
+          {% endcomment %}
+          {% for entry in site.data.contracts.entries %}
+            {% assign entry_footnotes = entry.footnotes | join: "," | split: "," %}
+            {% if entry_footnotes contains footnote_id_str %}
+              {% unless products contains entry.product %}
```

Comment:
So this `{% unless %}` guards against adding the same product twice, when multiple product entries have the same footnote?

Reply:
Yeah, unfortunate that they don't have a way of doing `not contains`.